test/dummies/vars.go

```diff
@@ -171,7 +171,7 @@ func SetDummyCAPCClusterVars() {
 	CSClusterKind = "CloudStackCluster"
 	CSClusterName = "test-cluster"
 	EndPointHost = "EndpointHost"
-	EndPointPort = int32(8675309)
+	EndPointPort = int32(5309)
```

Comment:
TBH, kidna sad that 7 digits is too much for a real port here. I _do_ like my easter eggs.